imap_processing/lo/l2/lo_l2.py

```diff
@@ -0,0 +1,169 @@
+"""IMAP-Lo L2 data processing."""
+
+import numpy as np
+import xarray as xr
+
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+from imap_processing.ena_maps import ena_maps
+from imap_processing.ena_maps.ena_maps import RectangularSkyMap
+from imap_processing.spice import geometry
+from imap_processing.spice.geometry import SpiceFrame
+
+
+def lo_l2(sci_dependencies: dict, anc_dependencies: list) -> list[xr.Dataset]:
+    """
+    Will process IMAP-Lo L1C data into Le CDF data products.
+
+    Parameters
+    ----------
+    sci_dependencies : dict
+        Dictionary of datasets needed for L2 data product creation in xarray Datasets.
+    anc_dependencies : list
+        Ancillary files needed for L2 data product creation.
+
+    Returns
+    -------
+    created_file_paths : list[Path]
+        Location of created CDF files.
+    """
+    # create the attribute manager for this data level
+    attr_mgr = ImapCdfAttributes()
+    attr_mgr.add_instrument_global_attrs(instrument="lo")
+    attr_mgr.add_instrument_variable_attrs(instrument="enamaps", level="l2-common")
```

Comment:
You also need the "l2-rectangular" variable attributes to get the DEPEND_i and LABL_PTR_i values.
```suggestion
    attr_mgr.add_instrument_variable_attrs(instrument="enamaps", level="l2-common")
    attr_mgr.add_instrument_variable_attrs(instrument="enamaps", level="l2-rectangular")
```